source/js/components/join/join.jsx

```diff
@@ -124,23 +124,28 @@ export default class JoinUs extends React.Component {
     });
   }
 
+  createClasslist(classes = ``) {
```

Comment:
This helper method can be removed once https://github.com/mozilla/foundation.mozilla.org/issues/2959 is landed.